usermods/ULC_Battery_Management/IP5306_I2C.h

```diff
@@ -0,0 +1,584 @@
+/*@brief  library to interact with IP5306 over I2C communication
```

Comment:
Instead of using a copy of the driver library, could you replace this file with a reference to library on github?

Something like 
```
lib_deps = .... 
    github_user/IP5306 @ 0.1.23
```

this would be a better solution, because we would avoid having a full copy of the driver - that might become outdated in a year or two - as part of WLED.

Reply:
The library is modified to work with the IP5306 used here. There is no library on github for it as of now.